docs/changes.rst

```diff
@@ -40,6 +40,13 @@ Released: not yet
 
 **Enhancements:**
 
+* Test: Added support for testing from Pypi/GitHub source distribution archives.
```

Comment:
Suggestion: very short why this is important here would help the reader. i.e. One sentence

Reply:
DONE - Added this sentence:
```
  This allows testing without having to check out the entire repository, and
  is convenient for testing e.g. when packaging pywbem into OS-level packages.
```